pkg/scaling/cache/scalers_cache.go

```diff
@@ -138,6 +138,7 @@ func (c *ScalersCache) GetMetricsAndActivityForScaler(ctx context.Context, index
 	}
 	startTime = time.Now()
 	metric, activity, err = ns.GetMetricsAndActivity(ctx, metricName)
+	log.Info("Get latency from GetMetricsAndActivityForScaler:", "latency", time.Since(startTime).Milliseconds(), "metricName", metricName, "err", err)
```

Comment:
Why this change please?

Reply:
I just added logs temporarily for checking purposes, but the test passed after I added logs. It seems like the latency metric is intermittent.